ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,281 @@
+    - Battery cell voltage bindings
+    - Battery driver support
+    - BattEsimate.lua applet estimates SoC from voltage
+    - Camera and Mount bindings improved
+    - CAN input packet filtering reduces work required by Lua CAN drivers
+    - DJI RS2/RS3 gimbal driver supports latest DJI firmware version (see mount-djirs2-driver.lua)
+    - EFI drivers for DLA serial, InnoFlight Inject EFI driver
+    - EFI bindings improved
+    - Fence support
+    - Generator drivers for Halo6000, Zhuhai SVFFI
+    - GCS failsafe support
+    - Hobbywing_DataLink driver (see Hobbywing_DataLink.lua)
+    - is_landing, is_taking_off bindings
+    - led_on_a_switch.lua sets LED brightness from RC switch
+    - MAVLink sending and receiving support
+    - Mission jump_to_landing_sequence binding
+    - mount-poi.lua upgraded to applet, sends better feedback, can lock onto Location
+    - Networking/Ethernet support
+    - Proximity driver support
+    - Rangefinder drivers can support signal quality
+    - revert_param.lua applet for quickly reverting params during tuning
+    - RockBlock.lua applet supports setting mode, fix for battery voltage reporting
+    - Serial/UART reading performance improvement using readstring binding
+    - sport_aerobatics.lua rudder control fixed
+    - Thread priority can be set using SCR_THD_PRIORITY (useful for Lua drivers)
+    - Wind alignment and head_wind speed bindings
+13) Safety related enhancements and fixes
+    - Arm/Disarmed GPIO may be disabled using BRD_OPTIONS
+    - Arming check of compass vs world magnetic model to detect metal in ground (see ARMING_MAGTHRESH)
+    - Arming check of GPIO pin interrupt storm
+    - Arming check of Lua script CRC
+    - Arming check of mission loaded from SD card
+    - Arming check of Relay pin conflicts
+    - Arming check of emergency stop skipped if emergency stop aux function configured
+    - Arming failures reported more quickly when changing from success to failed
+    - ARMING_OPTIONS allows supressing "Armed", "Disarmed" text messages
+    - BRD_SAFETY_MASK extended to apply to CAN ESCs and servos
+    - Buzzer noise for gyro calibration and arming checks passed
+    - Dijkstras object avoidance supports "fast waypoints" (see AVOID_OPTIONS)
+    - FENCE_OPTIONS supports union OR intersection of all polygon fences
+    - FLTMODE_GCSBLOCK blocks GCS from changing vehicle to specified modes
+    - GCS failsafe action to switch to Brake mode (see FS_GCS_ENABLE)
+    - Main loop lockup recovery by forcing mutex release (only helps if caused by software bug)
+    - Rally points supports altitude frame (AMSL, Relative or Terrain)
+    - SERVO_RC_FS_MSK allows outputs using RC passthrough to move to SERVOx_TRIM on RC failsafe
+    - TKOFF_RPM_MAX aborts takeoff if RPM is too high (for cases where a propeller has come off)
+14) System Enhancements
+    - CAN port can support a second CAN protocol on the same bus (2nd protocol must be 11 bit, see CAN_Dn_PROTOCOL2)
```

Comment:
Support for STM32H730

Reply:
Hi @andyp1per,

Txs again for the review.  Users don't really know what an STM32H730 is for the most part though.  I figure this new CPU will appear in some autopilot and that will be the moment it will mean something to them.